tests/utils.py

```diff
@@ -1,32 +1,95 @@
 import json
-from typing import List, Dict
+from typing import List, Dict, Union, Type
 
+from src.entities.entity import Entity
+from src.entities.dataset import Dataset
 from src.entities.pipeline import Pipeline
+from src.entities.pipeline_run import PipelineRun
+from src.entities.problem.problem import Problem
 
-test_pipeline_names: List[str] = [
-    "simple_pipeline_a",
-    "simple_pipeline_b",
-    "pipeline_with_subpipelines",
-    "simple_pipeline_b_one_off",
-]
+_test_entity_names: Dict[str, Dict[str, Union[Type[Entity], List[str]]]] = {
+    'pipelines': {
+        'type': Pipeline,
+        'names': [
+            'simple_pipeline_a',
+            'simple_pipeline_b',
+            'pipeline_with_subpipelines',
+            'simple_pipeline_b_one_off',
+            'similar_pipeline_a',
+            'similar_pipeline_b',
+            'similar_pipeline_c',
+            'similar_pipeline_d',
+        ]
+    },
+    'pipeline_runs': {
+        'type': PipelineRun,
+        'names': [
+            'similar_pipeline_run_a',
+            'similar_pipeline_run_b',
+            'similar_pipeline_run_c',
+            'similar_pipeline_run_d',
+            'invalid_pipeline_run_a', # Has a state of FAILURE
+            'invalid_pipeline_run_b', # Has no predictions
+            'invalid_pipeline_run_c', # Phase is FIT, not PRODUCE
+            'invalid_pipeline_run_d', # Problem task type is unsupported
```

Comment:
This documentation of what each test data document contains and can be used for is great. Would you be willing to add a small comment like this to the similar pipeline runs and invalid problem as well?